core/config.go

```diff
@@ -71,6 +71,8 @@ type BodyConfig struct {
 	MaxLength *int64 `yaml:"maxLength,omitempty" default:"no max"`
 	// Block allows multiline text inputs for body messages
 	UseBlock bool `yaml:"block,omitempty" default:"false"`
+	// Edit body with text editor, if true then the body will be opened with a text editor
+	EditBodyWithTextEditor bool `yaml:"editBodyWithTextEditor,omitempty" default:"false"`
```

Comment:
This option is per use of new by using `--editor` so we should not add this config.

Reply:
How should askPromts method will know that --editor flag was used?
Or should I take the input before asking for the prompt?

Reply:
You can add it to the prompt context struct here https://github.com/miniscruff/changie/blob/fdb6f021920011ef1e652753a0a21b12efd4c4e8/core/change.go#L119, which you can add a boolean to AskPrompts for now to get it there. It will just be a one off extra boolean parameter which may look a little strange but we can refactor it later if we need to.

Reply:
Ok

Reply:
We are not passing the value of `n.BodyEditor' to the ` change.AskPrompts(config, n.InOrStdin())` so how will the contexts `bodyEditor` field should be changed?

Reply:
Right, I think its ok to add a parameter to AskPrompts for the body editor boolean.

Reply:
https://github.com/miniscruff/changie/blob/dd29fe056bfa9808b1b1429f2fd018ed0b05dfcc/core/change_test.go#L161C1-L161C43
should i hard code `false` to every test case?

Reply:
Declaring false is fine, will need to find a way to test the editor stuff though, not sure how hard that will be.

Reply:
As this involves another program the testing cases will be different.
I can't really think of any good ideas to do that..

Reply:
Hmm, I think for testing we could create our own interface of `Run() error` which the Cmd is using. Then when writing a test we can create our own mock Command to test valid and error cases. And in the real code we could use `os.ExecCommand`. And then we would just need to play with the functions and config structs for this to be feasible.

Reply:
Going to move this discussion to a new place as it went off a little bit